apps/user-office-backend/src/auth/StfcUserAuthorization.ts

```diff
@@ -44,6 +44,10 @@ export class StfcUserAuthorization extends UserAuthorization {
     Tokens.InstrumentDataSource
   );
 
+  protected userDataSource: StfcUserDataSource = container.resolve(
+    Tokens.UserDataSource
+  ) as StfcUserDataSource;

```

Comment:
I don't think it's the best practice to use type assertion here, but I opted to do this over adding the `isSearchableUser` function to the `UserDataSource` interface. I think in this case we can always reasonably expect that the STFC auth would use the STFC data source, but let me know if the other way is better practice.

Reply:
Looks good to me